app/views/admin/worldwide_organisation_pages/index.html.erb

```diff
@@ -1,8 +1,4 @@
-<% content_for :back_link do %>
-  <%= render "govuk_publishing_components/components/back_link", {
-    href: admin_worldwide_organisations_path,
-  } %>
-<% end %>
+
```

Comment:
...and can you replace your commit message with essentially what you've put in your PR description?
(More useful to have it in the commit too, so that tools like IDEs can surface that info, whereas PR descriptions aren't stored in the git history and are tied only to GitHub.com)